tests/phpunit/TemplateTest.php

```diff
@@ -214,9 +214,11 @@ public function testRemoveWikilinks() {
     $this->assertEquals('Pure Evil', $expanded->get('title-link'));
     $expanded = $this->process_citation("{{Cite journal|title=[[Dark]] Lord of the [[Sith (Star Wars)|Sith]] [[Pure Evil]]}}");
     $this->assertEquals('Dark Lord of the Sith Pure Evil', $expanded->get('title'));
-  
+    $expanded = $this->process_citation("{{Cite journal|title=Dark Lord of the [[Sith (Star Wars)|Sith]] Pure Evil}}");
+    $this->assertEquals('Dark Lord of the Sith Pure Evil', $expanded->get('title'));
+    $this->assertEquals('Sith (Star Wars)', $expanded->get('title-link'));
```

Comment:
Is this really the expected behaviour?  The link refers to part of the title, not the full title, so the user is unlikely to expect the full title to link to the partial link.